pkg/piecestore/psserver/psdb/psdb.go

```diff
@@ -103,6 +103,84 @@ func (db *DB) Migration() *migrate.Migration {
 					)`,
 				},
 			},
+			{
+				// some databases may have already this done, although the version may not match
+				Description: "Optimize storagenode tables",
```

Comment:
I would change description to something more meaningful like `Extending bandwidth_agreements table`

Reply:
updated the table description

Reply:
updated